cloudbuild.yaml

```diff
@@ -3,6 +3,6 @@ options:
   substitution_option: ALLOW_LOOSE
   machineType: E2_HIGHCPU_32
 steps:
-- name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud
+- name: gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20240718-5ef92b5c36
```

Comment:
stick to a tag  https://github.com/kubernetes-sigs/kube-network-policies/pull/71#discussion_r1705670459

/assign @aojea @ameukam